app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -90,9 +93,11 @@ class LoginActivity : AppCompatActivity(), ILoginView {
     }
 
     override fun onLoginError(title: String?, message: String?) {
+        progressDialog.dismiss()
```

Comment:
yes I have latest code. Yesterday I was facing this issue regularly. Closing it for now . will open it if I face this issue